### PR TITLE
[EDU-5231] Axur Cardstream guide

### PR DESCRIPTION
**Pull request by guiafonso-ol**
### Changes

Adds the Axur Cardstream integration guide, already fit to the new version of the function.

### CODE REVIEW

File: src/content/docs/en/pages/guides/marketplace/integrations/cardstream.mdx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+<Tabs client:visible>
+    <Fragment slot="tab.console">Console</Fragment>
+    <Fragment slot="tab.rtm">Real-Time Manager</Fragment>
+
+<Fragment slot="panel.console">
+1. On the upper-left corner, open the **Products menu** and select **Edge Firewall** in the **Secure** section.
+2. Click the **+ Edge Firewall** button.
+3. Give an easy-to-remember name to your edge firewall.
+4. Select the domains you want to protect with the function.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/cardstream.mdx:78]: We need to check this step. I'll send you a DM

File: src/content/docs/en/pages/guides/marketplace/integrations/cardstream.mdx [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+  "api_key": "YourAxurAPIKey",
+}
+```
+
+Where:
+
+| Variable | Mandatory | Description |
+| ---- | ---- | ---- |
+| `card_number_field` | Yes | The number of the credit card used in the transaction. |
+| `api_key` | Yes | API Key of your Axur account. |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/cardstream.mdx:131]:
```suggestion
| `card_number_field` | Yes | The number of the credit card used in the transaction |
| `api_key` | Yes | The API Key of your Axur account |
```

File: src/content/docs/en/pages/guides/marketplace/integrations/cardstream.mdx [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+1. Select the **Rules Engine** tab.
+2. Click the **+ Rule Engine** button.
+3. Give an easy-to-remember name to the rule.
+4. Select a *criteria* to run and catch the domains that you want to run the integration on.
+  - Example: if `Host` *matches* `yourdomain.com`.
+5. Below, select the **Run Function** *behavior* to the *criteria*.
+6. Select the **Axur Cardstream** function according to the name you gave it before.
+7. Click the **Save** button.
+
+Done. Now the **Upstash Cardstream** integration is running.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/cardstream.mdx:185]:
```suggestion
Done. Now the **Axur Cardstream** integration is running.
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/cardstream.mdx [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+
+<Tabs client:visible>
+    <Fragment slot="tab.console">Console</Fragment>
+    <Fragment slot="tab.rtm">Real-Time Manager</Fragment>
+
+<Fragment slot="panel.console">
+1. No canto superior esquerdo, abra o **Products menu** e selecione **Edge Firewall** na seção **Secure**.
+2. Clique no botão **+ Edge Firewall**.
+3. Dê um nome fácil de lembrar ao seu edge firewall.
+4. Selecione os domínios que deseja proteger com a função.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/cardstream.mdx:78]: idem

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/cardstream.mdx [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+  "api_key": "YourAxurAPIKey",
+}
+```
+
+Onde:
+
+| Variável | Obrigatório | Descrição |
+| ---- | ---- | ---- |
+| `card_number_field` | Sim | O número do cartão de crédito usado na transação. |
+| `api_key` | Sim | A chave da API da sua conta Axur. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/cardstream.mdx:131]:
```suggestion
| `card_number_field` | Sim | O número do cartão de crédito usado na transação |
| `api_key` | Sim | A chave da API da sua conta Axur |
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/cardstream.mdx [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/cardstream.mdx:158]:
```suggestion
| `card_number_field` | Sim | O número do cartão de crédito usado na transação |
| `api_key` | Sim | A chave da API da sua conta Axur |
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/cardstream.mdx [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+
+6. Clique no botão **Save**.
+</Fragment>
+
+</Tabs>
+
+### Configure uma regra no Rules Engine
+
+Para concluir, você deve configurar uma regra no **Rules Engine** para configurar os critérios (*criteria*) e o comportamento (*behavior*) para executar a função.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/cardstream.mdx:167]:
```suggestion
Para concluir, você deve configurar uma regra no **Rules Engine** para definir os critérios (*criteria*) e o comportamento (*behavior*) para executar a função.
```

**Review by MarianaAguilera — APPROVED** [review]

**Review by GabrielAzion — APPROVED** [review]

**Pull request merged**